src/containers/Header/Header.scss

```diff
@@ -8,25 +8,24 @@
 
     padding: 0 20px 0 12px;
 
-    color: var(--g-color-text-primary);
     border-bottom: 1px solid var(--g-color-line-generic);
 
-    a {
-        color: var(--g-color-text-secondary);
-
-        &:hover {
-            color: var(--g-color-text-complementary);
-        }
-    }
+    &__breadcrumbs {
+        &_item {
+            display: flex;
+            align-items: flex-start;
+            gap: 3px;
 
-    &__breadcrumb {
-        display: flex;
-        align-items: center;
+            color: var(--g-color-text-secondary);
 
-        &__icon {
-            display: flex;
+            &_icon {
```

Comment:
In our code, we try to avoid more than one level of nesting (except for modifiers). This means that entities should follow each other and not be enclosed.